packages/smart-account/src/configs.ts

```diff
@@ -0,0 +1,60 @@
+import * as common from '@protocolink/common';
+
+export enum SmartAccountId {
+  PORTUS = '1',
+}
+
+export interface SmartAccountExecutor {
+  id: string;
+  address: string;
+}
+
+export interface Config {
+  chainId: number;
+  executors: SmartAccountExecutor[];
+}
+
+export const configs: Config[] = [
+  {
+    chainId: common.ChainId.optimism,
+    executors: [
+      {
+        id: SmartAccountId.PORTUS,
+        address: '0xdD408b8eFb837EdeF8e6192Ed19f0dbEB7B79383',
+      },
+    ],
+  },
+];
```

Comment:
```suggestion
export const configs: Config[] = [
  {
    chainId: common.ChainId.optimism,
    smartAccounts: [
      {
        id: SmartAccountId.PORTUS,
        executor: '0xdD408b8eFb837EdeF8e6192Ed19f0dbEB7B79383',
      },
    ],
  },
];
```

Looks like the object key is more like smart accounts as the `PORTUS` is smart account id rather than executor's id.